httpbp/server.go

```diff
@@ -157,6 +157,21 @@ type ServerArgs struct {
 	// Logger is an optional arg to be called when the InjectEdgeRequestContext
 	// middleware failed to parse the edge request header for any reason.
 	Logger log.Wrapper
+
+	// UpstreamIssueLoggingSampleRates decides the sample rates to keep the logs
+	// emitted by stdlib http.Server regarding an upstream issue.
+	//
+	// The key is the upstream issue id.
+	// Currently the only supported one is 25192 [1],
+	// but the supported issue ids might expand in the future.
+	//
+	// When a supported issue id is not in the map,
+	// currently the default sample rate is 1
+	// (100% sampled, e.g. not dropping those logs).
+	// This might be changed to 0 in the future.
```

Comment:
🔕 I'm in kindof a YAGNI mode here... this is a lot of complexity to carry around for something we might not need.